src/main/java/Storage.java

```diff
@@ -0,0 +1,135 @@
+import java.io.*;
+import java.io.BufferedReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    protected static ArrayList<Task> taskList = new ArrayList<Task>();
+    protected static String file = "todo.txt";
+
+    public Storage(String file) {
+
+    }
+
+    /**
+     * Adds tasks to the file.
+     *
+     * @param filepath File that the task is added to.
+     * @param textToAdd Tasks that needs to be added.
+     * @throws IOException If the named file exists but
+     * is a directory rather than a regular file,
+     * does not exist but cannot be created, or
+     * cannot be opened for any other reason.
+     */
+    public void addToFile(String filepath, String textToAdd) throws IOException {
+            FileWriter typer = new FileWriter(filepath, true);
+            typer.write(textToAdd + System.lineSeparator());
```

Comment:
Have to say that I didn't know about this `System.lineSeparator()` function. Looks like it might be useful for a cross-platform application. Good job 👍👍